src/celeritas/grid/NonuniformGridBuilder.hh

```diff
@@ -6,58 +6,41 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+#include <utility>
+#include <vector>
+
+#include "corecel/Types.hh"
 #include "corecel/data/Collection.hh"
-#include "corecel/data/CollectionBuilder.hh"
 #include "corecel/data/DedupeCollectionBuilder.hh"
 #include "corecel/grid/NonuniformGridData.hh"
-#include "corecel/grid/SplineDerivCalculator.hh"
+#include "celeritas/Types.hh"
 #include "celeritas/inp/Grid.hh"
 
 namespace celeritas
 {
 //---------------------------------------------------------------------------//
 /*!
- * Construct a generic grid.
- *
- * This uses a deduplicating inserter for real values to improve cacheing.
+ * Construct a nonuniform grid.
```

Comment:
Was there a reason to delete the text about deduplication, or do all our builders do that now?

Reply:
Not intentional, I must have just gotten carried away with the deleting. But yes, all our grid inserters should be using deduplicating builders.